test-cases/longevity/longevity-large-partition-2days.yaml

```diff
@@ -0,0 +1,43 @@
+test_duration: 3000
+
+bench_run: true
+
+prepare_write_cmd:  ["scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=100 -clustering-row-count=50000 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=100 -clustering-row-count=50000 -partition-offset=100 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
```

Comment:
All the partition offsets are worng.
It must be 101, 201, 301, 401, 501, 601 and 701.
For now, you redefine last one by next loader and do not write last, 800th partition.
It will cause failures.

Reply:
fixed